### DISCUSSION TITLE
Under review for a long time, and no response

### DISCUSSION THREAD

**Discussion by vickenliu** [WalletConnect/walletconnect-monorepo#4500] [Developer Support]
some details i found for your reference
[
    {
        "id": "03fa72ca-af6f-46c0-b703-5aeb34772531",
        "project_uuid": "e91f4ed5-de84-424b-8c91-52c9bbae3dd9",
        "created_at": "2024-02-06T20:56:48.060907+00:00",
        "updated_at": "2024-02-06T20:56:48.060907+00:00"
    }
]

Can you please help us to track what is going on with our submission? Thank you.

**Comment by Cali93** [comment]
Hey @vickenliu,

Explorer listings for dapps is being deprecated. We don't review and approve dapps anymore.
We will update the Cloud dashboard shortly to reflect that.
